app/assets/javascripts/map/gmap.js.erb

```diff
@@ -133,14 +132,13 @@ function loadScript() {
   $("body").on("submit", "#save-route-form", function(event){
     event.preventDefault();
     var url = $(this).attr("action")
-    var total_miles = (total * 0.000621371192).toFixed(2)
+    var total_miles = document.getElementById('total').innerHTML
```

Comment:
@ifricker @ajpage @michaelzwang @ficxin 
You are using jQuery for most of this code, the `document.getElementById` call seems out of place since it is interacting with the document directly instead of going through jQuery. I would like to see this be consistent and converted to use jQuery.